recipes-core/core-framework/redhawk_2.2.2.bb

```diff
@@ -23,20 +23,20 @@ require core-framework-autotools.inc
 DESCRIPTION = "REDHAWK Core Framework"
 
 PR = "4"
+PREFERRED_PROVIDER_virtual/python-threading ?= "bulkiointerfaces"
```

Comment:
This is a non-functional change; please remove. (Bitbake packages can't modify one another in this way.)